internal/attestation/claim.go

```diff
@@ -46,12 +51,17 @@ func parseClaims(claims []C.oe_claim_t) (Report, error) {
 			report.TCBStatus = tcbstatus.Status(claimUint(claim))
 		case C.OE_CLAIM_SGX_REPORT_DATA:
 			report.Data = claimBytes(claim)
+		case C.OE_CLAIM_SGX_TCB_INFO:
+			tcbInfo = claimBytes(claim)
+		case C.OE_CLAIM_SGX_TCB_INFO_INDEX:
+			tcbInfoIndex = claimUint(claim)
 		}
 	}
 
 	if !hasAttributes {
 		return Report{}, errors.New("missing attributes in report claims")
 	}
+	report.TCBAdvisories, report.TCBAdvisoriesErr = getAdvisoriesFromTCBInfo(tcbInfo, tcbInfoIndex)
```

Comment:
Will the tcbInfo and tcbIndex fields also be populated if attestation passes successfully?
I guess in that case one also just wouldn't need to check the advisories in the first cases, so should be fine either way

Reply:
Yes, they will, but `advisoryIDs` should be empty like in the first level of the test.